src/components/CoverButtons.jsx

```diff
@@ -0,0 +1,13 @@
+import heart from "../assets/icons/heart.svg";
+import dots from "../assets/icons/dots.svg";
+import play from "../assets/icons/play.svg";
+
+export const CoverButtons = () => {
+  return (
+    <div className="CoverButtonsDiv fade">
```

Comment:
Cool that you did the intermediate stretch goal and got the play button to have full opacity